src/sonic-bgpcfgd/tests/test_allow_list.py

```diff
@@ -0,0 +1,296 @@
+from app.allow_list import BGPAllowListMgr
+from app.directory import Directory
+from app.template import TemplateFabric
+
+from mock import MagicMock, patch
+import pytest
+
+
+def new_run_command(cmd):
+    return 0, "", ""
+
+global_constants = {
+    "bgp": {
+        "allow_list": {
+            "enabled": True,
+            "default_pl_rules": {
+                "v4": [ "deny 0.0.0.0/0 le 17" ],
+                "v6": [
+                    "deny 0::/0 le 59",
+                    "deny 0::/0 ge 65"
+                ]
+            }
+        }
+    }
+}
+
+#@pytest.mark.skip()
+@patch('app.allow_list.run_command', side_effect=new_run_command)
+def test_set_handler_with_community(mock_run_command):
```

Comment:
can you generate the test coverage data and upload? would like to check the new code coverage here.

Reply:
Done.